src/main/resources/db/changelog/changes/v0001.sql

```diff
@@ -0,0 +1,44 @@
+CREATE SEQUENCE IF NOT EXISTS customer_details_seq
+    INCREMENT 50
+    START 1;
+
+CREATE SEQUENCE IF NOT EXISTS ride_details_seq
+    INCREMENT 50
+    START 1;
+
+CREATE SEQUENCE IF NOT EXISTS taxi_details_seq
+    INCREMENT 50
+    START 1;
+
+CREATE TABLE IF NOT EXISTS customer_details

```

Comment:
group sequence and table as one and split this file into 3, also try to add xml format for learning.